src/config.ts

```diff
@@ -38,7 +38,7 @@ const fileNames = [
 export function stripTemplateFromFileName(filePath: string) {
 	const path = parse(filePath);
 	path.base = path.base.replace('.template', '');
-	return format(path);
+	return normalize(format(path));
```

Comment:
Without normalizing the path we get something like `/foo/bar/\\.gitignore` which is kind of weird, but I think technically works.